### PR TITLE
Fix code with black

### PR DESCRIPTION
**Pull request by jakirkham**
Fix CI command running `black` and rerun `black` on codebase.

Checklist
* [x] Added a ``news`` entry

### CODE REVIEW

**Comment by CJ-Wright** [comment]
@jakirkham did black not need to change anything?

**Comment by jakirkham** [comment]
It does need to fix things. Am trying to make sure the CI test is working correctly before committing them.

**Comment by CJ-Wright** [comment]
@jakirkham is this ready?

File: conda_smithy/_version.py [hunk 1]
```diff
@@ -1,4 +1,3 @@
-
 # This file helps to compute a version number in source trees obtained from
```
Comment (isuruf) [inline conda_smithy/_version.py]: Can you remove black from running on this file? This is a vendored file I believe.

Comment (jakirkham) [reply conda_smithy/_version.py]: It's installed by versioneer. That said, why not update its style as well?

**Comment by jakirkham** [comment]
Should be ready (pending any review comments). 👍

File: conda_smithy/ci_register.py [hunk 2]
```diff
@@ -50,9 +50,7 @@
     anaconda_token = os.environ["BINSTAR_TOKEN"]
 except KeyError:
     try:
-        with open(
-            os.path.expanduser("~/.conda-smithy/anaconda.token"), "r"
-        ) as fh:
```
Comment (isuruf) [inline conda_smithy/ci_register.py]: Last change was by black in https://github.com/conda-forge/conda-smithy/commit/b4ea9c39676644ff83945c0a84ff91ea76f490a7 .
Why did black decide to do this earlier and now is reverting it?

Comment (CJ-Wright) [reply conda_smithy/ci_register.py]: I might have put in the pep8 line length flag in last time 🐑

Comment (jakirkham) [reply conda_smithy/ci_register.py]: Ok added that flag to CI and re-ran with it. Also squashed all black commits to avoid adding noise to the history.

**Comment by jakirkham** [comment]
Ok should be ready for another review. 🙂

**Comment by jakirkham** [comment]
Does that +1 mean you are happy to merge @CJ-Wright? 😉

**Comment by CJ-Wright** [comment]
Yes! Sorry I was happy before I thought you were waiting for others

**Comment by jakirkham** [comment]
> Yes! Sorry I was happy before I thought you were waiting for others

I was waiting for Isuru's response, but he has since responded. 🙂

**Comment by CJ-Wright** [comment]
Now I'm confused 😕

**Comment by jakirkham** [comment]
> Now I'm confused 😕

He marked this thread ( https://github.com/conda-forge/conda-smithy/pull/1168#pullrequestreview-298377433 ) as resolved.

**Pull request merged**